ff_assignment_portal/ff_assignment_portal/doctype/ff_assignment_submission/ff_assignment_submission.py

```diff
@@ -64,10 +64,31 @@ def enqueue_generate_similarity_score(self):
 		frappe.enqueue_doc(
 			ASSIGNMENT_DOCTYPE_NAME,
 			self.name,
-			"_generate_similarity_score",
+			"_generate_similarity_score_alt",
 			queue="long",
 		)
 
+	@frappe.whitelist()
+	def _generate_similarity_score_alt(self):
+		submissions = frappe.db.get_all(
+			self.doctype,
+			filters={"day": self.day, "status": "Passed", "user": ("!=", self.user)},
+		)
+		
+		max_similarity_score = 0
+		similar_assignment = None
+		current_user_file_contents = self.get_filename_with_contents() #{filename: filecontent}
+
+		for submission in submissions:
+			other_user_file_contents = submission.get_filename_with_contents()
```

Comment:
You can't directly call a method on the result of get all, it is a dict. You have to first do get_doc.